src/features/MovieList/MovieTile/styled.js

```diff
@@ -1,125 +1,158 @@
 import styled from "styled-components";
 import {ReactComponent as Star} from "../../../images/star.svg";
 
-export const Content = styled.div`
+export const Content = styled.li`
     padding: 16px;
     background-color: ${({ theme }) => theme.color.white};
     box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
     border-radius: 5px;
     display: flex;
     flex-direction: column;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
         flex-direction: row;
     }
 `;
 
 export const Img = styled.img`
-    aspect-ratio: 2/3;
+    width: 100%;
+    height: 100%;
     border-radius: 5px;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         width: 50%;
     }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        width: 40%;
+    }
 `;
 
 export const Wrapper = styled.div`
     display: flex;
     flex-direction: column;
     height: 100%;
-    justify-content: space-between;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         margin-left: 16px;
-        width: 50%;
-        display: flex;
-        justify-content: flex-start;
     }
 `;
 
-export const DescriptionContainer = styled.div`
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
-          
-        }
-`;
-
 export const Title = styled.h3`
-    margin: 16px 0px 8px;
+    margin: 16px 0 8px;
     font-weight: 500;
     font-size: 22px;
     line-height: 130%;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
         font-size: 16px;
+        margin: 0 0 4px;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        font-size: 4vw;
```

Comment:
If you are specifying font-size in vw units you should also specify margins and paddings in vw units. It does not look good when font is getting smaller, while the spacings remain the same. In order to resolve this issue you could either get rid of those vw units overall or add all of the margins and paddings in vw units also. I know i have done this with all the margins in my latest branch for backdrop description, but setting up all of the components like that is a huge overkill and definitely too time consuming. Also we should carefully use vw units, and by carefully i mean when it really is a good idea.